media_driver/linux/common/os/mos_os_specific.c

```diff
@@ -1288,13 +1293,27 @@ MOS_STATUS Linux_InitContext(
     // when MODS enabled, intel_context will be created by pOsContextSpecific, should not recreate it here, or will cause memory leak.
     if (!MODSEnabled)
     {
-       pContext->intel_context = mos_gem_context_create(pOsDriverContext->bufmgr);
+        pContext->intel_context = mos_gem_context_create_ext(pOsDriverContext->bufmgr,0);
+        if (pContext->intel_context)
+        {
+            pContext->intel_context->vm = mos_gem_vm_create(pOsDriverContext->bufmgr);
+            if (pContext->intel_context->vm == nullptr)
+            {
+                MOS_OS_ASSERTMESSAGE("Failed to create vm.\n");
+                return MOS_STATUS_UNKNOWN;
+            }
+        }
+        else //try legacy context create ioctl if DRM_IOCTL_I915_GEM_CONTEXT_CREATE_EXT is not supported
+        {
+           pContext->intel_context = mos_gem_context_create(pOsDriverContext->bufmgr);
+           pContext->intel_context->vm = nullptr;
+        }
 
-       if (pContext->intel_context == nullptr)
-       {
-            MOS_OS_ASSERTMESSAGE("Failed to create drm intel context");
-            return MOS_STATUS_UNKNOWN;
-       }
+        if (pContext->intel_context == nullptr)
+        {
+             MOS_OS_ASSERTMESSAGE("Failed to create drm intel context");
+             return MOS_STATUS_UNKNOWN;
+        }
```

Comment:
This block looks like just whitespace noise.

Reply:
Yeah, original source indented with 3 spaces.  To make the patch clear, I will keep it unchanged in this commit.